superset/views/core.py

```diff
@@ -2126,13 +2126,15 @@ def csv(self, client_id):
             columns = [c['name'] for c in obj['columns']]
             df = pd.DataFrame.from_records(obj['data'], columns=columns)
             logging.info("Using pandas to convert to CSV")
-            csv = df.to_csv(index=False, encoding='utf-8')
+            csv = df.to_csv(index=False, encoding=str(config.get('CSV_ENCODING')),
```

Comment:
why calling str() on the options?

Reply:
To re-cast the string, otherwise I get `TypeError: "delimiter" must be string, not unicode` with python 2.x 
If you have a better solution, I can change that (or maybe add comment to clarify inside the code ?)

Reply:
Should be fine then